coly-api.md

```diff
@@ -73,7 +73,7 @@ Our data models have several types(or layers) that define our approach. Simply s
 Any model could be either public or private:
 
 - Public means it's accessible from outside(any API consumers like: frontend applications, client services, widgets, etc.).
-- Private means that such a model would be unreachable from outside and used solely by system internals. Usually used to extend [primary models](#primary-models) with computed data.
+- Private means that such a model would be unreachable from outside and used solely by system internals. Usually used to extend [primary models](#primary-models) with computed data. // Why do we share this? 
```

Comment:
this added comment obviously needs to be deleted before merging, but wanted to know if there was any idea behind sharing private models?

Reply:
Shouldn't be present in here

Reply:
I removed the entire text chunk that says:

```
Any model could be either public or private:

- Public means it's accessible from outside(any API consumers like: frontend applications, client services, widgets, etc.).
- Private means that such a model would be unreachable from outside and used solely by system internals. Usually used to extend [primary models](#primary-models) with computed data.
```